project-base/app/tests/FrontendApiBundle/Functional/Price/HiddenPricesTest.php

```diff
@@ -99,4 +100,30 @@ public function testCustomerUserCannotSeeOrderPrices(): void
             $this->assertSame('***', $item['unitPrice']['priceWithoutVat']);
         }
     }
+
+    public function testCustomerUserCannotSeeGatewayPayments(): void
+    {
+        $query = '
+            query {
+                payments {
+                    uuid
+                    type
+                    price {
+                        priceWithVat
+                        priceWithoutVat
+                    }
+                }
+            }
+        ';
+
+        $response = $this->getResponseContentForQuery($query);
+        $payments = $response['data']['payments'];
+        $this->assertCount(4, $payments);
+
+        foreach ($payments as $payment) {
+            $this->assertSame('***', $payment['price']['priceWithVat']);
+            $this->assertSame('***', $payment['price']['priceWithoutVat']);
```

Comment:
`MoneyFormatterHelper::HIDDEN_FORMAT` could be used here, right?

Reply:
I can change it, but it was on a purpose. If format is changed it will have an effect on GQL output. FE counts on that format. So, when someone modifies it, tests should fail from my point of view. What do you think ?